### PR TITLE
Add history handler tests

### PR DESCRIPTION
**Pull request by racheljay**
This PR adds a test for the handler function for the `/api/history` endpoint.

### CODE REVIEW

File: api/test_history_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+//go:build medium
```
Comment (KyleJu) [inline api/test_history_test.go:1]: For medium tests, the medium keyword should be specified in the file name, e.g. [labels_medium_test.go](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/labels_medium_test.go)

File: api/test_history_test.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	ctx, done, err := sharedtest.NewAEContext(true)
+	assert.Nil(t, err)
+	defer done()
+
+	sampleRun := shared.TestHistoryEntry{
+		BrowserName: "chrome",
+		RunID:       "123",
+		Date:        "2022-06-02T06:02:55.000Z",
+		TestName:    "test name",
+		SubtestName: "",
```
Comment (KyleJu) [inline api/test_history_test.go:33]: Let's add a subtest name here

File: api/test_history_test.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		Status:      "PASS",
+	}
+
+	body :=
+		`{
+			"testName": "test name"
```
Comment (KyleJu) [inline api/test_history_test.go:39]: the naming convention for payload should be [snake case](https://github.com/web-platform-tests/wpt.fyi/tree/main/api#apisearch), e.g. test_name.

File: api/test_history_test.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+						"status": "PASS",
+						"run_id": "123",
+					},
+				},
+			},
+		},
+	}
+
+	assert.Equal(t, want, results)
+
```
Comment (KyleJu) [inline api/test_history_test.go:68]:
```suggestion
```

**Comment by KyleJu** [comment]
Looks great! Since test is written, don't forget an [API documentation](https://github.com/web-platform-tests/wpt.fyi/tree/main/api#readme) for public APIs

File: api/test_history_test.go (hunk 2 above)
Comment (racheljay) [reply api/test_history_test.go]: done 👍

File: api/test_history_test.go (hunk 3 above)
Comment (racheljay) [reply api/test_history_test.go]: done 👍

File: api/test_history_test.go (hunk 1 above)
Comment (racheljay) [reply api/test_history_test.go]: I decided to change it to be a small test. I'm wasn't sure what the differences between test sizes were.

**Comment by racheljay** [comment]
> Looks great! Since test is written, don't forget an [API documentation](https://github.com/web-platform-tests/wpt.fyi/tree/main/api#readme) for public APIs

Should I capture that in this PR, or make a new one?

**Comment by KyleJu** [comment]
> > Looks great! Since test is written, don't forget an [API documentation](https://github.com/web-platform-tests/wpt.fyi/tree/main/api#readme) for public APIs
> 
> Should I capture that in this PR, or make a new one?

Feel free to send a follow-up PR!

**Review by KyleJu — APPROVED** [review]

**Pull request merged**